src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -78,69 +61,29 @@ export type Props = {
 
 const gene_image_width = Number(settings.gene_image_width);
 
-const addProteinDomains = (
-  transcript: ProteinCodingTranscript,
-  proteinDomains: ProteinDomain[]
-) => {
-  return set(
-    ['product_generating_contexts', '0', 'product', 'protein_domains'],
-    proteinDomains,
-    transcript
-  ) as TranscriptWithProteinDomains;
-};
-
 const ProteinsListItemInfo = (props: Props) => {
   const { gene, transcript, trackLength } = props;
   const params = useParams<'genomeId' | 'entityId'>();
   const { genomeId } = params;
 
-  const [transcriptWithProteinDomains, setTranscriptWithProteinDomains] =
-    useState<TranscriptWithProteinDomains | null>(null);
-
   const [proteinSummaryStats, setProteinSummaryStats] = useState<
     ProteinStats | null | undefined
   >();
 
-  const [domainsLoadingState, setDomainsLoadingState] = useState<LoadingState>(
-    LoadingState.LOADING
-  );
-
   const [summaryStatsLoadingState, setSummaryStatsLoadingState] =
     useState<LoadingState>(LoadingState.LOADING);
 
   const { trackProteinDownload } = useEntityViewerAnalytics();
 
-  const proteinId =
-    transcript.product_generating_contexts[0].product.unversioned_stable_id;
-
-  const { product: productWithProteinDomains } =
-    transcriptWithProteinDomains?.product_generating_contexts[0] || {};
+  const productId = transcript.product_generating_contexts[0].product.stable_id;
 
   const proteinXrefs = getProteinXrefs(transcript);
   const displayXref = proteinXrefs[0];
 
-  useEffect(() => {
-    const abortController = new AbortController();
-
-    if (domainsLoadingState === LoadingState.LOADING) {
-      fetchProteinDomains(proteinId, abortController.signal)
-        .then((proteinDomains) => {
-          if (!abortController.signal.aborted) {
-            setTranscriptWithProteinDomains(
-              addProteinDomains(transcript, proteinDomains)
-            );
-            setDomainsLoadingState(LoadingState.SUCCESS);
-          }
-        })
-        .catch(() => {
-          setDomainsLoadingState(LoadingState.ERROR);
-        });
-    }
-
-    return function cleanup() {
-      abortController.abort();
-    };
-  }, [domainsLoadingState]);
+  const { data } = useProteinDomainsQuery({
+    productId: productId,
+    genomeId: params.genomeId as string
```

Comment:
`params.genomeId` -> just `genomeId` (it's already declared above).

Reply:
Done